src/pages/Main/weeklybest/Product.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import './Product.scss';
+
+class Product extends Component {
+  render() {
+    return (
+      <div className="product-info">
+        <img alt="product_img" clasName="img-origin" src={this.props.img} />
+        <img
+          alt="mainslide_img"
+          className="img-hover"
+          src="images/Main/leaf.jpg"
```

Comment:
```suggestion
          src="/images/Main/leaf.jpg"
```
`/` 붙여주세요, 안붙일경우 현재 경로에서 상대적으로 접근을 해버립니다!
`src="images/logo.png"`
예를들어 public/images/logo.png를 가지고 오고 싶어서 위와 같이 작성을 해뒀는데,
localhost:3000/detail/1에서 위 img tag를 불러오게되면
현재 url에서 상대젹인 위치인 `public/detail/images/logo.png` 로 접근을 하게 됩니다.
당연히 이미지가 안뜨겠죠? img src를 작성할 때는 꼭 `/` 적어서 절대경로 명시해주세요!